rapthor/lib/screen.py

```diff
@@ -334,17 +334,25 @@ def fit(self):
         vals = [[ra*np.pi/180.0, dec*np.pi/180.0] for ra, dec in zip(ra_deg, dec_deg)]
         sourceTable = list(zip(*(soltab_ph.dir, vals)))
 
-        # Now call LoSoTo's stationscreen operation to do the fitting
+        # Now call LoSoTo's stationscreen operation to do the fitting. For the phase
+        # screens, we reference the the phases to the station with the least amount
+        # of flagged solutions, drawn from the first 10 stations (to ensure it is
+        # fairly central)
```

Comment:
Is it guaranteed that the stations with the lowest index are also the most central?

Reply:
With LOFAR at least I think it is (at least in my experience).